src/components/githubRepoCard/GithubRepoCard.scss

```diff
@@ -54,8 +68,6 @@
 }
 
 .repo-name {
-  white-space: nowrap;
-  text-overflow: ellipsis;
```

Comment:
Yeah, because of changing to flex, ellipsis will not work. If we want ellipsis, then we need to make some more changes.

Reply:
I tried it and it breaks the layout into a long single card (which is probably how it was before your changes too). 
We can change the layout container to use grid or add a max width to the card so it doesn't extend into a single long card. This would also solve the ellipsis problem as the container would now have a specified width so the p tag can just be `width: 100%` now

![image](https://user-images.githubusercontent.com/48270786/190894430-d07225c6-2ffc-4508-995c-00c22c77a194.png)

Reply:
Sure I will do that and push!